examples/layer-browser/data/pattern.json

```diff
@@ -0,0 +1,30 @@
+{
+  "hatch-1x": {
+    "x": 4,
+    "y": 4,
+    "width": 120,
+    "height": 120,
+    "mask": true
```

Comment:
does this extension support per object mask?

Reply:
Not at the moment. I am trying to reduce the number of attributes that are added by this extension so that it can be used with more layers.